kola/tests/ignition/mount.go

```diff
@@ -0,0 +1,331 @@
+// Copyright 2019 Red Hat
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ignition
+
+import (
+	"strings"
+
+	"github.com/coreos/mantle/kola/cluster"
+	"github.com/coreos/mantle/kola/register"
+	"github.com/coreos/mantle/platform/conf"
+)
+
+func init() {
+	// verify mount to `/var/lib/containers`
+	register.Register(&register.Test{
+		Name:        "coreos.ignition.mount.var-lib-containers",
```

Comment:
Hmm, do we actually need to split this out into three separate tests instead of just the combined one?

Reply:
The Jira card stated it as three different tests so I thought we wanted three separate ones here, though I can't see any reasons not to just make a combined one.

Reply:
Yeah, I think it'd be cleaner. WDYT about having two tests:

- one where we create the two partitions on the primary disk
- one where we attach two additional disks, one for each partition

?

Reply:
SGTM :+1: